src/app/clusters/network-commissioning/NetworkCommissioningCluster.cpp

```diff
@@ -253,7 +253,7 @@ void NetworkCommissioningCluster::SetLastNetworkingStatusValue(
 {
     if (mLastNetworkingStatusValue.Update(networkingStatusValue))
     {
-        NotifyAttributeChanged(Attributes::LastNetworkingStatus::Id);
+        NotifyAttributeChanged(Attributes::LastNetworkingStatus::TypeInfo::GetAttributeId());
```

Comment:
Why this change? It results in the same Id since that is a return of the same value ... and this makes the code just longer. Lets not do this.

```suggestion
        NotifyAttributeChanged(Attributes::LastNetworkingStatus::Id);
```

Reply:
The change in the other direction makes sense to me (shorter code, clearer).